test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -367,27 +405,61 @@ std::string RandomCode::fillArguments(dev::eth::Instruction _opcode, RandomCodeO
 }
 
 
-//Ramdom Code Options
-RandomCodeOptions::RandomCodeOptions() : useUndefinedOpCodes(false), smartCodeProbability(50)
+//Default Ramdom Code Options
+RandomCodeOptions::RandomCodeOptions() : useUndefinedOpCodes(false), smartCodeProbability(100)
 {
 	//each op code with same weight-probability
 	for (auto i = 0; i < 255; i++)
-		mapWeights.insert(std::pair<int, int>(i, 50));
+		mapWeights.insert(std::pair<int, int>(i, 40));
+
 	setWeights();
+
+	//Probability of instructions
+	setWeight(eth::Instruction::STOP, 1);
+	for (int i = (int)(eth::Instruction::PUSH1); i < 32; i++)
+		setWeight((eth::Instruction) i, 1);
+	for (int i = (int)(eth::Instruction::SWAP1); i < 16; i++)
+		setWeight((eth::Instruction) i, 10);
+	for (int i = (int)(eth::Instruction::DUP1); i < 16; i++)
+		setWeight((eth::Instruction) i, 10);
+
+	setWeight(eth::Instruction::SSTORE, 70);
+	setWeight(eth::Instruction::CALL, 70);
+	setWeight(eth::Instruction::CALLCODE, 70);
+	setWeight(eth::Instruction::DELEGATECALL, 70);
+	setWeight(eth::Instruction::STATICCALL, 70);
+	setWeight(eth::Instruction::EXTCODECOPY, 70);
+	setWeight(eth::Instruction::EXTCODESIZE, 70);
+
+	//some smart addresses for calls
+	addAddress(Address("0xffffffffffffffffffffffffffffffffffffffff"));
```

Comment:
Instead of these `push_back`s you could initialize this vector with initializer list in braces { .. }
```
RandomCodeOptions::RandomCodeOptions(): 
  useUndefinedOpCodes(false), 
  smartCodeProbability(100),
  addressList{Address("0xffffffffffffffffffffffffffffffffffffffff"), Address("0x1000000000000000000000000000000000000000"), ...}
```